contrib/hawq-docker/README.md

```diff
@@ -83,6 +81,38 @@ Type "help" for help.
 
 postgres=# 
 ```
+# Store docker images in local docker registry
+
+After your hawq environment is up and running, you could draft a local docker registry to store your hawq images locally for further usage.
+* pull and run a docker registry
+```
+docker pull registry
+docker run -d -p 127.0.0.1:5000:5000 registry
```

Comment:
Another 127.0.0.1, I guess it can be change to localhost too?

Reply:
I could not use "localhost"  here, i have tried to use no ip address instead.
FYI.

```
root@localhost incubator-hawq]# docker run -d -p localhost:5000:5000 registry
/usr/bin/docker-current: Invalid ip address: localhost.
See '/usr/bin/docker-current run --help'.
[root@localhost incubator-hawq]# docker run -d -p 5000:5000 registry
40758a8d59f51b6ae345595f71e7787e2245cdbe48cad61c024196c94fbdab4c
[root@localhost incubator-hawq]# docker ps
CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                    NAMES
40758a8d59f5        registry                 "/entrypoint.sh /e..."   7 seconds ago       Up 6 seconds        0.0.0.0:5000->5000/tcp   nifty_shirley
629ab67b77e8        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode3
60ac39e610b7        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode2
6ba58cfaf591        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode1
b6e65bc29227        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-namenode
[root@localhost incubator-hawq]#
```